packages/amplify-e2e-core/src/utils/rds.ts

```diff
@@ -76,6 +92,7 @@ export const createRDSInstance = async (config: RDSConfig): Promise<{ endpoint:
       endpoint: dbInstance.Endpoint.Address as string,
       port: dbInstance.Endpoint.Port as number,
       dbName: dbInstance.DBName as string,
+      dbInstance,
```

Comment:
Added dbInstance to support discovery of VPC configs elsewhere. Technically we could just return the dbInstance since endpoint, port, dbName are all contained therein, but that was more work than I wanted to take on since it would impact existing tests.